TicketsDemo/App_Start/NinjectWebCommon.cs

```diff
@@ -72,7 +72,10 @@ private static void RegisterServices(IKernel kernel)
 
             kernel.Bind<ISchedule>().To<Schedule>();
             kernel.Bind<ITicketService>().To<TicketService>();
-            kernel.Bind<IReservationService>().To<ReservationService>();
+            //kernel.Bind<IReservationService>().To<ReservationService>();
+
+            kernel.Bind<IReservationService>().To<ReservationServiceLoggingDecorator>();
+            kernel.Bind<IReservationService>().To<ReservationService>().WhenInjectedExactlyInto<ReservationServiceLoggingDecorator>();
```

Comment:
don't see using of HolidayPriceCalculationStrategy anywhere in app as it is not declared in IoC